### PR TITLE
Updated TablePoint.vue file to use Ant Design Tables

### LINKED ISSUE #248: Refactor: Update `TablePoint.vue` file to use Ant Design Tables
### Is your feature request related to a problem? Please describe.

Currently the [`TablePoint.vue`](flint.ui\src\views\flint\TablePoint.vue) file is implemented with raw CSS and doesn't fit with latest UI design.

We will be using the Table component at multiple places in configurations for data display of interfaces.

### Describe the solution you'd like.

Refactor the `TablePoint.vue` file by replacing the css with Table Ant Design component and complying with our Table Design.

### Describe alternatives you've considered

_No response_

### Additional context.

Refer: https://antdv.com/components/table/#Table

### PR DESCRIPTION
**Pull request by Janvi-Thakkar**
## Description
Fixes #248 
Updated TablePoint.vue file to use Ant Design Tables.
Added PointOuterTable.vue file to allow switching between graph and table
Added Table and graph on configuration page itself as suggested in the Figma design.

## Testing
1. Explore the Point page from the homepage of flint-ui
2. Click on run to load the point data.
3. Click on the Point Output tabel to see Table and Graph.

If our automated tests fail, please feel free to reach out to us by leaving a comment down below and we will be happy to take a look

image
image
I could also pass the props of row and column directly from the point page but it might be complex the information. Also, I can further modify it to make it complete reusable

### CODE REVIEW

**Comment by Janvi-Thakkar** [comment]
@chicken-biryani @sohamsshah can you please review this?

**Comment by chicken-biryani** [comment]
@SanjaySinghRajpoot @YashKandalkar

File: flint.ui/src/views/flint/PointOuterTable.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            <div class="full-w">
+                <PointOutput v-if="!isTable" />
+                <Table v-else :columns="columns" :rows="rows" />
+            </div>
+        </div>
+</template>
+
+<script>
+
+    /* eslint-disable */
```
Comment (SanjaySinghRajpoot) [inline flint.ui/src/views/flint/PointOuterTable.vue:16]: specific the exact eslint disable command

**Comment by SanjaySinghRajpoot** [comment]
Kindly sign your commits and fix merge conflicts, so far the changes look good to me. Once you are done with the remaining changes I will review it again.

**Comment by Janvi-Thakkar** [comment]
@SanjaySinghRajpoot I am done with the changes. Kindly review it!

**Comment by SanjaySinghRajpoot** [comment]
sign is still missing, run the below commands.  
- In your local branch, run: `git rebase HEAD~4 --signoff`
- Force push your changes to overwrite the branch: `git push --force-with-lease origin tablepoint-antd`

else Changes look good to me!!

**Review by SanjaySinghRajpoot — APPROVED** [review]

**Comment by Janvi-Thakkar** [comment]
> Great work, LGTM! @Janvi-Thakkar Please resolve the conflicts and we are good to be merged.

Thank You:) I tried resolving conflict already and succeed but when I did a force push it again showed conflict because of issues in the dependency tree as my master branch was behind flint-UI's master branch. Please suggest the needful!

**Comment by Janvi-Thakkar** [comment]
> ok, so your main branch is not updated, right? To fix this issue just head over to your main branch -> run `git pull` cmd, this will update the main branch. Now make a new branch in which you will make the changes. All the required changes are already available in this PR just replicate them on your new branch, close this PR make a new one from that branch. This is one of the simplest solution that will probably fix your problem.

Had the same thought but was unsure about closing the PR and raising the new one in this case. Thank you for suggesting the same.

**Pull request closed without merging**